circle.yml

```diff
@@ -1,3 +1,7 @@
+machine:
+  node:
+    version: 6.1.0
```

Comment:
`setup_linux` seems to use v7.x of Node, while CircleCI will be using v6.x. It might be a good idea to make it consistent?